packages/renderer/src/ffmpeg-flags.ts

```diff
@@ -1,12 +1,13 @@
 import execa from 'execa';
-import {binaryExists} from './validate-ffmpeg';
+import {validateFfmpeg} from './validate-ffmpeg';
 
 let buildConfig: string | null = null;
 
 const getFfmpegBuildInfo = async () => {
 	if (buildConfig !== null) {
 		return buildConfig;
 	}
+	await validateFfmpeg();
```

Comment:
Unfortunately this function calls `process.exit(1)`.
Functions in `renderer` should never call this, only from the CLI. Otherwise we are at risk crashing servers to whom Remotion is just one of many dependencies. Reverting this and merging, hope you understand!